packages/sitecore-jss/src/editing/graphql-editing-service.ts

```diff
@@ -118,13 +118,13 @@ export class GraphQLEditingService {
   async fetchEditingData({
```

Comment:
Update [jsdoc](https://github.com/Sitecore/jss/pull/1835/files#diff-f789af9cd3c7fecbd14a81c93a869ffd706d0804b9ce442b04ad7fd533590909R114), now it became optional: